authors/apps/articles/views.py

```diff
@@ -129,3 +143,110 @@ def destroy(self, request, *args, **kwargs):
         super().destroy(self, request, *args, **kwargs)
 
         return Response({'message': 'The article has been deleted.'})
+
+
+class CommentAPIView(ListCreateAPIView):
```

Comment:
I don't think we need to have two classes to create a `Comment` you can use a `ListModelMixin` to provide the listing functionality to the second class

Reply:
please kindly clarify for me, do you mean i should combine the two classes i created for the comment? @esirK